client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/common/Parameters.kt

```diff
@@ -20,4 +20,5 @@ internal object MigrationParameters {
     const val SOURCE_PASSWORD = "Password for the BioStudies user in the source environment"
     const val TARGET_USER = "BioStudies user in the target environment"
     const val TARGET_PASSWORD = "Password for the BioStudies user in the target environment"
+    const val TARGET_OWNER = "Optional new owner the submission in the new environment"
```

Comment:
I'd suggest: "Optional new owner for the submission in the target environment"

Reply:
or "New owner for the submission in the target environment (Optional)" :)